src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/service/RecallService.kt

```diff
@@ -99,7 +98,7 @@ class RecallService(
       assessedByUserId = updateRecallRequest.assessedByUserId?.value ?: assessedByUserId,
       bookedByUserId = updateRecallRequest.bookedByUserId?.value ?: bookedByUserId,
       dossierCreatedByUserId = updateRecallRequest.dossierCreatedByUserId?.value ?: dossierCreatedByUserId,
-      dossierTargetDate = calculateDossierTargetDate(updateRecallRequest.recallNotificationEmailSentDateTime)
+      dossierTargetDate = calculateInCustodyDossierTargetDate(inCustodyRecallOrBeingUpdatedToBeElseNull(updateRecallRequest), updateRecallRequest.recallNotificationEmailSentDateTime)
```

Comment:
Feels like even a very thin `calculateDossierTargetDate()` would still help this be clearer.